jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -156,6 +161,9 @@ node {
             } else {
                 cmd << "--auto-previous"
             }
+            if (params.TRIGGER_BUILD_SYNC) {
+                cmd << "--auto-trigger-build-sync"
+            }
 
             buildlib.withAppCiAsArtPublish() {
                 withCredentials([string(credentialsId: 'art-bot-slack-token', variable: 'SLACK_BOT_TOKEN'), string(credentialsId: 'openshift-bot-token', variable: 'GITHUB_TOKEN')]) {
```

Comment:
We need credentials set 'JENKINS_SERVICE_ACCOUNT' and 'JENKINS_SERVICE_ACCOUNT_TOKEN'

Reply:
Added and repushed.